server/src/word-list.ts

```diff
@@ -0,0 +1,127 @@
+import {
+    TextDocument,
+    CompletionItem,
+    CompletionItemKind,
+    TextDocumentChangeEvent
+} from 'vscode-languageserver';
+
+// A data structure to keep track of the words used in this file
+// We currently use a set, can be changed to a trie for more efficient lookup
+class WordListClass {
+
+    // The keywords 
+    keywords : Set<string>;
```

Comment:
Note that TypeScript fields are public by default (these look like they should be private)